biz.aQute.bndlib/src/aQute/bnd/osgi/repository/BridgeRepository.java

```diff
@@ -122,13 +154,7 @@ private synchronized void init() {
 		}
 
 		public InfoCapability getInfo() {
-			List<Capability> capabilities = resource.getCapabilities("bnd.info");
-			InfoCapability info;
-			if (capabilities.size() >= 1) {
-				info = ResourceUtils.as(capabilities.get(0), InfoCapability.class);
-			} else
-				info = null;
-			return info;
+			return BridgeRepository.getInfo(resource);
```

Comment:
Since this is the BridgeRepository class, do you need to name the class before the static method name?

Reply:
Get an error otherwise ...